SetupDataPkg/Tools/WindowBinToNvram.py

```diff
@@ -0,0 +1,148 @@
+# @file
+#
+# Set a dumpbin formated set of variables to NVRAM
+#
+# Copyright (c), Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+import os
+import sys
+import logging
+import argparse
+import struct
+import uuid
+import ctypes
+from SettingSupport.UefiVariablesSupportLib import UefiVariable  # noqa: E402
+
+gEfiGlobalVariableGuid ="8BE4DF61-93CA-11D2-AA0D-00E098032B8C"
+
+def option_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-guid",
+        "--guid",
+        dest="InputGuid",

```

Comment:
What is this input for? I thought the input bin already has GUID included?

Reply:
Will Remove.